CrossPlatformDesktopProject/CrossPlatformDesktopProject/IBatState.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
From what I could tell, this interface is not implemented anywhere and is no longer needed.